src/main/kotlin/no/nav/helsearbeidsgiver/domene/inntektsmelding/v1/utils/FeiletValidering.kt

```diff
@@ -21,6 +21,8 @@ internal object Feilmelding {
     const val KREVER_BELOEP_STOERRE_ENN_NULL = "Beløp må være større enn 0"
     const val AGP_IKKE_TOM = "Arbeidsgiverperioden må fylles ut, med mindre man betaler redusert lønn i perioden"
     const val AGP_MAKS_16 = "Arbeidsgiverperioden kan være maksimum 16 dager"
+    const val AGP_UNDER_16_OG_IKKE_BEHANDLINGSDAGER =
```

Comment:
Kanskje denne skal hete AGP_UNDER_16_OG_IKKE_GYLDIGE_BEHANDLINGSDAGER bare så det er lettere å skjønne koden - de kan jo ha *forsøkt å legge inn behandlingsdager. Trenger ikke å gjøre noe med selve feilmeldingen tror jeg, selv om det på en måte "mangler" en feilkode / forklaring for 12 ugyldige behandlingsdager.